sites/www.batdetective.org.conf

```diff
@@ -4,25 +4,10 @@ server {
 
     location /talk-player/ {
         resolver 8.8.8.8;
-        proxy_pass             http://zooniverse-static.s3-website-us-east-1.amazonaws.com/$host$request_uri;
-        # Exclude X-Frame-Options header
-        add_header 'Access-Control-Allow-Origin' '*';
-        add_header 'Access-Control-Allow-Credentials' 'true';
-        add_header 'Access-Control-Allow-Methods' 'GET';
-        add_header 'Access-Control-Allow-Headers' 'DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type';
-        add_header 'X-Content-Type-Options' 'nosniff';
-        add_header 'X-XSS-Protection' '1; mode=block';
-
+        proxy_pass             https://zooniversestatic.z13.web.core.windows.net/$host$request_uri;
         proxy_set_header       Host zooniverse-static.s3-website-us-east-1.amazonaws.com;
```

Comment:
wont' this clash with the proxy headers in `az-proxy-headers.conf`?

Reply:
Those lines should have both changed, yeah, but it wouldn't clash with anything in the az header file now. I'll update this now.